[FIX] 채팅방생성시 URL에 영향 줄수 있는 특수문자들 입력 불가하게 처리

Jake1152 opened the pull request:
- 채팅방생성시 URL에 영향 줄수 있는 특수문자들 입력 불가하게 처리 ( / \ % ? #)
- 채팅 입력 300자 제한

Jake1152 committed:
[FIX]

- 채팅방생성시 URL에 영향 줄수 있는 특수문자들 입력 불가하게 처리 ( / \
  % ? #)
- 채팅 입력 300자 제한

jinwoole reviewed (dismissed):
여러 가지 다른 방법이 있지만 뭐 되면 되는거 아니겠습니까
하지만 문구는 바꾸는게 좋겠습니다.

jinwoole commented:
merge는 안할거임. 대안 고려해보길 바람

Jake1152 committed:
[FIX]

- 기능 수정
	- AS IS
	  채팅방 생성시 URL에 영향주는 특수문자를 입력 못받게하고
	  alert으로 알려줌
	- TO BE
	  한글, 영어, 숫자, 공백문자 이외에는 입력조차 안되게 변경.

jinwoole reviewed (approved):
역시 짐님이 최고입니다. 그런데 사실 사용자가 /누른 상태로 입력해버리면 input에 보이진 않더라도 마지막에 버퍼까지 같이 들어갈 것만 같다는 의심이 듭니다

Jake1152 commented:
> 역시 짐님이 최고입니다. 그런데 사실 사용자가 /누른 상태로 입력해버리면 input에 보이진 않더라도 마지막에 버퍼까지 같이 들어갈 것만 같다는 의심이 듭니다

음 그런 악성유저는 크흠 마동석 프사라도 해야하나

Jake1152 commented:
> > 역시 짐님이 최고입니다. 그런데 사실 사용자가 /누른 상태로 입력해버리면 input에 보이진 않더라도 마지막에 버퍼까지 같이 들어갈 것만 같다는 의심이 듭니다
> 
> 음 그런 악성유저는 크흠 마동석 프사라도 해야하나

다행히 그런 경우는 발생하지 않았습니다
on:input이 잘 동작하는거 같네요
